x/committee/types/committee.go

```diff
@@ -13,32 +13,60 @@ import (
 
 const MaxCommitteeDescriptionLength int = 512
 
-// ------------------------------------------
-//				Committees
-// ------------------------------------------
+type TallyOption int
+
+const (
+	FirstPastThePost TallyOption = iota // 0
+	Deadline         TallyOption = iota // 1
+)
+
+const (
```

Comment:
Add descriptive comments for each committee type

Reply:
https://github.com/Kava-Labs/kava/pull/899/commits/5ff4eeb6591dcf2040bb900af67fdfe6b2165c98